### ISSUE TITLE
feat(PrintService): Pouvoir exporter des rapports (étape 4)

### ISSUE THREAD

**Issue by cboucheIGN** [IGNF/geoportail-environnement.beta.gouv.fr#9]
# Besoin

Pouvoir extraire les synthéses et rapport sous un document (pdf) à télécharger.

# Contrainte

Un rapport produit par un site DSFR doit aussi respecter le DSFR (il existe un design des rapports)

# Solution

Proposer une solution technique.

Si besoin créer un service en backend dédié à l'impression (puppetter) / voir implémentation dans projet ign-mut/igd/igd-api

**Comment by cboucheIGN** [comment]
A terme le parcours fichier impression ne sera peut être pas retenu.

Est-il possible de simuler le print pour préparer le fichier et l'envoyer en téléchargement ?

**Comment by David-IGN** [comment]
Quand tu dis le parcours fichier impression ne sera pas retenu.
 - Il n'y aura pas de bouton de prévisualisation de la page à imprimer ?

Pour "simuler" le print pour un envoie on a toujour la possibilité d'utiliser l'api symfony ou le service geportail-environement.beta.gouv.fr-printer. Mais niveau développement ca implique pas mal de problème : 

1 - Comment récupérer les informations pour la carte

- situé la carte
- niveau de zoom
- dessin réaliser par l'utilisateur (multi-polygone)
- couche afficher ou non avec leur opacité

Solution possible :
- Toutes les infos pourrai être passé par url

2 - Récupère les textes des différents onglet des thématiques

Solution possible :
- Les texts sont en base de donnée et il est possible des les récupères via api ?

**Comment by cboucheIGN** [comment]
Yes, du coup on garde l'idée d'une page impression prêt à print.

Dans un premier temps ce sera à l'utilistaeur de clicker sur "fichier -> impression"

Puis on pourra raffiner en demandant à une service de lancer l'impression en ouvrant cette page en lui passant en POST le contexte (foret + vue de la carte + liste des features interesté pour ne pas refaire les calculs d'intersection)
